src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -24,19 +24,19 @@ Users who need manual instrumentation should reference the "Datadog.Trace" packa
 
   <!-- For VS testing purposes only, copy all implementation assemblies to the
        to test the loading of managed profiler assemblies from disk -->
-  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and '$(TargetFramework)' == 'netstandard2.0'">
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and (( '$(TargetFramework)' == 'netstandard2.0' ) or (( '$(TargetFramework)' == 'netstandard2.1' ))) ">
```

Comment:
I'm not sure if the `netstandard2.1` changes are necessary since we're only building for `netstandard2.0` at the moment

Reply:
Correct. Good catch, meant to remove that.